components/tag-list/tag-list-item-mixin.js

```diff
@@ -0,0 +1,86 @@
+import '../colors/colors.js';
+import { css, html } from 'lit-element/lit-element.js';
+import { labelStyles } from '../typography/styles.js';
+
+export const TagListItemMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * @ignore
+			 */
+			role: { type: String, reflect: true },
+			/**
+			 * @ignore
+			 */
+			tabindex: { type: Number, reflect: true }
+		};
+	}
+
+	static get styles() {
+		return [labelStyles, css`
+			:host {
+				display: grid;
```

Comment:
How is the `grid` layout being used?  I don't see any other grid definition that I _think_ would be needed.

Reply:
So this was the only `display` value that would make the ellipsis truncating work properly with `flex`.  I unfortunately don't remember the details super well since I got all that working about a month and a half ago, but if I remove this the functionality completely breaks and the `nowrap` tags refuse to shrink and instead just overflow the container.  I remember trying a lot of combinations and this was the only one that worked, but I can try others if you have an idea!